scout/parse/models.py

```diff
@@ -282,7 +282,7 @@ class ScoutLoadConfig(BaseModel):
     gene_fusion_report: Optional[str] = None
     gene_fusion_report_research: Optional[str] = None
     gene_panels: Optional[List[str]] = []
-    genome_build: int = Field(38, alias="human_genome_build")
+    genome_build: str = Field(38, alias="human_genome_build")
```

Comment:
Here also, this comes from another branch

Reply:
Fixed.